### ISSUE TITLE
Tried to rob ye estate but me filled bags didn't fit the door frame.

### ISSUE THREAD

**Issue by lossycrypt** [sirenfal-factorio/Free_Real_Estate#4]
Entering the estate with a full inventory is impossible. Same goes for leaving it.

**Comment by avosirenfal** [comment]
Working as intended, yarr. This is unavoidable with my choice of "pick up item to enter". Sorry

**Comment by lossycrypt** [comment]
Is the event not fired at all in that case?

At least add a proper localized name to the pickup item. Currently the error message you get lists the usual unnamed garbage.

**Comment by avosirenfal** [comment]
Yeah, I don't believe it fires any events. Never thought of the localization problem, whoops. Thank you for helping, I really appreciate it.

I like your artful issue descriptions too. My friends and I had a laugh at it :P

**Comment by avosirenfal** [comment]
Incidentally I might add a "go near to enter" option at some point like Factorissimo, but I don't have time right now. I hated that because I kept entering factories by accident, and because you couldn't abuse SqueakThrough with factories stacked vertically right next to each other.

**Comment by lossycrypt** [comment]
Personally i haven't used factorissimo, but it seems trivial to add a delay to "stand near to enter". I.e. stand near the entrance for 2 seconds or something, that way ST should still work. As a workaround you could add a temporary item slot to the player if you see they have a full inventory. Tho that would require some longish detection code to not cause other issues i guess. If the key-item is stackable adding an extra slot an putting a not-full stack in the temporary slot would work.Might use an invisible item icon...dunno.

Glad you liked the titles. Not everyone likes jokes .P

**Comment by lossycrypt** [comment]
Oh also on exiting the factory if you stand near the outer corners of the exit you're not close enough to pick up the item anymore. Might be good to place some more or move it closer to the edge.

**Comment by lossycrypt** [comment]
Also another afterthought, if one checked for the actual player orientation on a "go near" imlementation there would be even less accidential entering.
